priv/kafka.bnf

```diff
@@ -250,3 +250,9 @@ DescribeGroupsResponse => [GroupDescription]
     MemberMetadata => bytes
     MemberAssignment => bytes
 
+SaslHandshakeRequest => Mechanism
+  Mechanism => string
+
+SaslHandshakeResponse => ErrorCode [EnabledMechanism]
+  ErrorCode => int16
+  EnabledMechanism => string
```

Comment:
but `[EnabledMechanism]` *is* an array of `enabled mechanism`s. I just tried to blend into the current code style.
Looks like it'd be the first plural in the whole file. Naming conventions probably could be changed in one pass (looks like your https://github.com/klarna/kafka_protocol/pull/21 does exactly that), but, ok, I'm changing it.